src/cares_wrap.cc

```diff
@@ -1246,6 +1246,9 @@ class QueryAnyWrap: public QueryWrap {
                                ret,
                                addrttls,
                                &naddrttls);
+    if (naddrttls > static_cast<int>(arraysize(addrttls))) {
+      naddrttls = arraysize(addrttls);
+    }
```

Comment:
minor nit but these could be replaced with `naddrttls = std::min(naddrttls, arraysize(addrttls)`